File: Application/src/main/java/com/example/android/viewholder/LoadingViewHolder.java
package com.example.android.viewholder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ProgressBar;

import com.example.android.recyclerview.R;

/**
 * Created by KNothing on 2017/12/6.
 */
public class LoadingViewHolder extends RecyclerView.ViewHolder  {
    public ProgressBar progressBar;
    public LoadingViewHolder(View itemView) {
        super(itemView);
        progressBar = (ProgressBar) itemView.findViewById(R.id.progressBar1);
    }
}

File: Application/src/main/java/com/example/android/viewholder/LoadingViewHolder.kt
package com.example.android.viewholder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ProgressBar

import com.example.android.recyclerview.R

/**
 * Created by KNothing on 2017/12/6.
 */
class LoadingViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var progressBar: ProgressBar

    init {
        progressBar = itemView.findViewById<View>(R.id.progressBar1) as ProgressBar
    }
}
